result/timelineGenerator.js

```diff
@@ -29,27 +36,82 @@ function addContainer(content) {
     section.classList.add('timelineSection')
 
 
-
     const contentDiv = document.createElement("div");
     contentDiv.classList.add("timelineContent")
     contentDiv.insertAdjacentHTML('beforeend', content.htmlContent);
+    contentDiv.style.height = contentHeight + 'px'
 
-    section.append(addIcon(content.label))
+    section.appendChild(addIcon(content.label))
     section.appendChild(contentDiv)
+
+    section.appendChild(addReadMore(content))
     eventBox.appendChild(section)
     event.appendChild(eventBox)
 
     return event
 }
 
+function addReadMore(content) {
+    const readMore = document.createElement('div')
```

Comment:
Maybe check for overflow here?
perhaps something like this would help https://stackoverflow.com/questions/143815/determine-if-an-html-elements-content-overflows
This probably wouldn't handle the changes in the screen size though, so this might need more thinking and perhaps a separate task